test/cli.js

```diff
@@ -0,0 +1,107 @@
+var chai = require('chai');
+var child = require('child_process');
+var util = require('./util');
+var exec = child.exec;
+var expect = chai.expect;
+
+describe('cli', function() {
+  var input = util.read('fixtures/cli/input');
+  var output = util.read('fixtures/cli/input.out');
+
+  afterEach(function() {
+    util.remove('fixtures/cli/output');
+  });
+
+  it('should read from a file and write to a file', function(done) {
+    exec('node bin/suitcss -c test/config/noautoprefixer.config.js test/fixtures/cli/input.css test/fixtures/cli/output.css', function(err) {
```

Comment:
Just a thought:
since we now support only Node v4 and above we could start to use some ES6 here.

e.g. I'd create an util for `exec`:
```javascript
const cli = (args, cb) => child.exec(`node bin/suitcss ${args.split(/\s+/).join(' ').trim()}`, cb)

cli(`
    -c test/config/noautoprefixer.config.js 
    test/fixtures/cli/input.css 
    test/fixtures/cli/output.css `, 
  err => {
  }
)
```

What do you think?